aerospike/assets/dashboards/overview.json

```diff
@@ -0,0 +1,1424 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "Aerospike Overview",
+    "created": "2019-12-24T20:03:21.354815+00:00",
+    "created_by": {
+        "access_role": "adm",
+        "disabled": false,
+        "email": "support@datadoghq.com",
+        "handle": "support@datadoghq.com",
+        "is_admin": true,
+        "name": "Datadog",
+        "role": null,
+        "verified": true
+    },
+    "description": "",
+    "disableCog": false,
+    "disableEditing": false,
+    "id": 932434,
+    "isIntegration": false,
+    "isShared": false,
+    "modified": "2019-12-25T00:51:13.359243+00:00",
+    "new_id": "4iq-sen-4xc",
+    "originalHeight": 80,
+    "originalWidth": "calc(100% - 32px)",
+    "read_only": false,
+    "template_variables": [
+        {
+            "default": "*",
+            "name": "scope",
+            "prefix": ""
+        }
+    ],
+    "widgets": [
```

Comment:
Since there are a lot of graphs, you can categorise them using labels with arrows.